iocore/net/OCSPStapling.cc

```diff
@@ -174,8 +209,31 @@ ssl_stapling_init_cert(SSL_CTX *ctx, X509 *cert, const char *certname)
   cinf->certname    = ats_strdup(certname);
   cinf->resp_derlen = 0;
   ink_mutex_init(&cinf->stapling_mutex);
-  cinf->is_expire   = true;
-  cinf->expire_time = 0;
+  cinf->is_prefetched = rsp_file ? true : false;
+  cinf->is_expire     = true;
+  cinf->expire_time   = 0;
+
+  if (cinf->is_prefetched) {
+    Note("Using OCSP prefetched response file %s", rsp_file);
```

Comment:
Hmmm, should we really use Note() here? That seems pretty chatty ? @randall  ?

Reply:
that's the easy way of telling if you're using prefetched OCSP responses or dynamic ones, but it can be replaced with Debug()

Reply:
I've switched it to Debug() already